src/components/biodiversity-sidebar-card/biodiversity-sidebar-card-component.jsx

```diff
@@ -10,6 +10,8 @@ import { biodiversityCategories } from 'constants/mol-layers-configs';
 import { BIODIVERSITY_TABS } from 'constants/ui-params';
 import styles from './biodiversity-sidebar-card-styles.module.scss';
 
+import BiodiversityThumbnail from "images/biodiversity.webp";
```

Comment:
I do not think we should use `webp` format just yet. ([check Caniuse table](https://caniuse.com/webp))
I would go with `png`that has [almost universal support.](https://caniuse.com/png-alpha)

Reply:
Mmm, I overlooked that safari 😓 . I will change that